terraform/globals.tf

```
@@ -69,9 +69,10 @@ variable "infra_subnet_ids" {
   default     = ""
 }
 
-variable "concourse_egress_cidr" {
-  description = "Public egress IP address of concourse running the pipeline"
-  default     = ""
+variable "concourse_egress_cidrs" {
+  description = "Set or unset the concourse_egress_cidrs"
```

I think this could be clearer. Perhaps something like:

```suggestion
  description = "Allow or restrict public egress IP address of concourse workers"
```

---

Agreed. I'll integrate this change with [Will's suggestion](https://github.com/alphagov/paas-bootstrap/pull/568#discussion_r1138697257)on the branch and re-push it.